src/components/Header/Navigation.jsx

```diff
@@ -0,0 +1,39 @@
+import { useAuth } from "../../hooks";
+import styled from "styled-components";
+import { NavLink } from "react-router-dom";
+
+const Link = styled(NavLink)`
+  display: inline-block;
+  padding: 16px 10px;
+
+  text-decoration: none;
+  font-weight: 500;
+  color: #000;
+
+  &.active {
+    color: #b92f2c;
+  }
+  &:hover,
+  &:focus {
+    text-decoration: underline;
+  }
+`;
+export const Navigation = () => {
+    const { isLoggedIn } = useAuth();
+  return (
+    <nav>
+
+      {isLoggedIn && (
+        <div>
```

Comment:
It`s not so important,  but It`ll be more semantic to use ul with li tags

Reply:
ok. thanks.